coffee-module/coffee-module-redisstream/src/main/java/hu/icellmobilsoft/coffee/module/redisstream/bootstrap/BaseRedisConsumerStarter.java

```diff
@@ -86,18 +95,33 @@
  */
 public class BaseRedisConsumerStarter {
 
+    protected static final String MANAGED_EXECUTOR_SERVICE_CORE_THREADS = "MANAGED_EXECUTOR_SERVICE_CORE_THREADS";
+    private static final int DEFAULT_MAX_THREAD_COUNT = 16;
+    public static final int SAFETY_BUFFER = 10;
+    public static final String POOL_DEFAULT_MAX_TOTAL = "pool.default.maxtotal";
+    private static final String REDIS_PREFIX = "coffee.redis";
+
     @Inject
     private Logger log;
 
     @Resource
     private ManagedExecutorService managedExecutorService;
 
     @Inject
-    private StreamGroupConfig config;
+    private StreamGroupConfig streamGroupConfig;
+
+    @Inject
+    private Config config;
 
     @Inject
     private BeanManager beanManager;
 
+    @Inject
+    private ManagedRedisConfig managedRedisConfig;
+
+    @Inject
+    private Event<RedisStreamMetricEventMessage> metricsEvent;
```

Comment:
Esetleg az új funkció bekerülhetne doksiba pl, hogy van egy ilyen event, és egy példa is lehetne a használatáról.